08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,28 @@
-/*Add CSS rules*/
\ No newline at end of file
+main{
+  display:flex;
+  font-family: sans-serif;
+  
+}
+
+img{
+  width:95%;
+}
+
+button{
+  border-right: 3px solid maroon;
+  background-color: white;
+  border-top: 1px solid lightgrey;
+  border-bottom: 1px solid lightgrey;
+  border-left: 1px solid lightgrey;
```

Comment:
Since these three are all the same, it might be easier to declare `border:1px solid lightgrey;` and then override just the right side with `border-right: 3px solid maroon;`. Fewer lines of code and easier to see what's going on.